frontend/src/views/Pages/Register.vue

```diff
@@ -174,7 +174,7 @@ export default {
         this.password,
       )
       if (result.success) {
-        this.$store.dispatch('createUser', {
+        this.$store.dispatch('login', {
```

Comment:
I think we do not need this. After register, the user is redirected to the thank page. There is a link to login. So way store a session id when the user has to login again and the receives as session id? Does not make sense to me.